src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -948,15 +951,37 @@ boolean forkGitHub(Channel channel, User sender, String owner, String repo, Stri
 
             // GitHub adds a lot of teams to this repo by default, which we don't want
             Set<GHTeam> legacyTeams = r.getTeams();
+            GHTeam newTeam = null;
 
             try {
-                getOrCreateRepoLocalTeam(org, r, maintainers.isEmpty() ? singletonList(user.getName()) : maintainers);
+                // add the repo owner if there are no users specified and the repo owner is a user (not an org)
+                if(maintainers.isEmpty() && user.getType().equalsIgnoreCase("user")) {
```

Comment:
Don’t you need to clear the list if it’s not a user? I assume it’s still populated from before?

Reply:
The list is a non mutable list if it is empty at this point, so it requires being reallocated as a mutable list.

Reply:
i misread the code it's fine